### PR TITLE
Cthalmann/canopy cover validator

### PR DESCRIPTION
**Pull request by cthalmann**
fixes issue #79 

## Changes in this PR.
- [X] Adds validator for est_canopy_cover
- [X] Adds test for validator

## Testing this PR.
1.   docker-compose run web bash
2.    cd streamwebs_frontend
3.    python manage.py test streamwebs/tests/models

### Expected Output.

```
Creating test database for alias 'default'...
..........................................................
----------------------------------------------------------------------
Ran 58 tests in 0.710s

OK
Destroying test database for alias 'default'...


```

@osuosl/devs

### CODE REVIEW

File: streamwebs_frontend/streamwebs/models.py [hunk 1]
```diff
@@ -477,7 +485,8 @@ class Canopy_Cover(models.Model):
                               related_name='south', null=True)
     west = models.ForeignKey(CC_Cardinal, on_delete=models.CASCADE,
                              related_name='west', null=True)
-    est_canopy_cover = models.PositiveIntegerField(default=0)
+    est_canopy_cover = models.PositiveIntegerField(default=0,
+                                                   validators=[validate_pH])
```
Comment (athai) [inline streamwebs_frontend/streamwebs/models.py]: Wrong validator is being called.

**Comment by Kennric** [comment]
Looks correct to me. +1

**Comment by athai** [comment]
Looks good! +1

**Comment by alxngyn** [comment]
+1

**Pull request merged**